fract4d_compiler/absyn.py

```diff
@@ -4,7 +4,6 @@
 
 from . import fracttypes
 
-
```

Comment:
Two lines here and before top-level functions are as per PEP 8:
https://www.python.org/dev/peps/pep-0008/#blank-lines